recipes/icu/bld.bat

```diff
@@ -0,0 +1,15 @@
+if "%ARCH%"=="32" (
```

Comment:
See the **Why don't you provide project files for my MSVC version (MSVC 2008, etc)?** section at http://userguide.icu-project.org/icufaq

The rabbit hole I went down was that I tried to get a toolchain other than cygwin to work.  I never succeeded, and eventually gave up.  This might be something to revisit with @mingwandroid and his efforts with MSYS2 when he has time.